staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -100,11 +100,25 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr admission.Attr
 	}
 
 	wg := sync.WaitGroup{}
-	errCh := make(chan error, len(relevantHooks))
+	errCh := make(chan error, 2*len(relevantHooks)) // double the length to handle extra errors for panics in the gofunc
 	wg.Add(len(relevantHooks))
 	for i := range relevantHooks {
 		go func(invocation *generic.WebhookInvocation, idx int) {
+			ignoreClientCallFailures := false
+			defer func() {
+				if r := recover(); r != nil {
+					if ignoreClientCallFailures {
+						// if failurs are supposed to ignored, ignore it
+						klog.Warningf("Panic calling webhook, failing open %v: %v", invocation.Webhook.GetName(), r)
```

Comment:
```
					admissionmetrics.Metrics.ObserveWebhookFailOpen(ctx, hook.Name, "validating")
```
is this needed, just by looking at the block in L155 that handles the same logic

Reply:
> is this needed? just by looking at the block in L155 that handles the same logic

possibly.  Not sure of whether we need different markers for panic over error.

Reply:
> possibly. Not sure of whether we need different markers for panic over error.

added tracking